### DISCUSSION TITLE
Pricing?

### DISCUSSION THREAD

**Discussion by phantomdenkali** [thunderclient/thunder-client-support#1285] [Q&A]
I'm an indie developer, team of 1 that uses github as source control. It looks like thunder has gone paid only? Is that accurate? I thought per the pricing page, I should be able to use the free version until my team expands?

**Comment by phantomdenkali** [comment]
i see that there are new collection run limits of 30 per month. I basically run this all the time after every major code change. Would request that number to be bumped up higher.

**Reply by rangav** [reply]
Hi @phantomdenkali 

The free version will have limited features from next month.

compare plans here
https://www.thunderclient.com/pricing#compare

Free version limits

- No git sync
- 2 collections(15 reps per collection)
- 1 Environment 
- 30 collection runs


Let me know your feedback

**Comment by phantomdenkali** [comment]
It seems to be prompting me to subscribe already. The 30 collection runs seems to be the limit I'm hitting. I run that number of runs daily  :(  All other limits seem OK, though the free edition does say github support is included?

**Reply by rangav** [reply]
Support means if you have any questions ask on GitHub
